### ISSUE TITLE
Synfig crash while color picking for a layer

### ISSUE THREAD

**Issue by AnishGG** [synfig/synfig#781]
**Synfig version & platform**:
- Latest master branch, commit [24824e9](https://github.com/synfig/synfig/commit/24824e9858ef429e591a316d8476c80ea2e9ae69)
- Ubuntu 18.04.2 LTS, 64 bit

**Issue description**:
- Create a new layer in synfig
- Open color change panel for that layer
- Click on the outer ring of the color picker and drag it
- -->Crash (Aborted (core dumped))
![synfig_issue](https://user-images.githubusercontent.com/30872212/54526950-6afaea00-499e-11e9-8469-cc0665a3bc90.gif)

**Comment by morevnaproject** [comment]
Hi @AnishGG ! Are you using debug or production build type?

**Comment by AnishGG** [comment]
I am using debug build type.

**Comment by morevnaproject** [comment]
Ah, I see. Just want to mention that in production build this doesn't happen.

**Comment by ankit-kumar-dwivedi** [comment]
I don't see any crash on Linux Mint 19
@morevnaproject I think this should be closed. [^_^]

**Comment by morevnaproject** [comment]
@ankit-kumar-dwivedi Did you tested on Debug build type? This error happens on Debug build only. ^__^

**Comment by ankit-kumar-dwivedi** [comment]
> @ankit-kumar-dwivedi Did you tested on Debug build type? This error happens on Debug build only. ^__^

yes I tried on debug build 2-3 times

**Comment by ankit-kumar-dwivedi** [comment]
this time it crashed with the following error
```
synfigstudio: /home/allievo/gsoc/synfig/synfig-studio/src/gui/workarearenderer/renderer_canvas.cpp:654: void studio::Renderer_Canvas::clear_render(): Assertion `enqueued_tasks == 0' failed.
Aborted (core dumped)
```

**Comment by morevnaproject** [comment]
> yes I tried on debug build 2-3 times

> this time it crashed with the following error

Looks like you tried on different branches.

It is possible the issue is fixed by this commit - 717751edcb711848bfcb1e181a3a399b3063e953
